src/pem.rs

```diff
@@ -421,6 +422,26 @@ pub enum Error {
     NoItemsFound,
 }
 
+impl fmt::Display for Error {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            Self::MissingSectionEnd { end_marker } => {
+                write!(f, "missing section end marker: {end_marker:?}")
+            }
+            Self::IllegalSectionStart { line } => {
+                write!(f, "illegal section start: {line:?}")
+            }
+            Self::Base64Decode(e) => write!(f, "base64 decode error: {e}"),
+            #[cfg(feature = "std")]
+            Self::Io(e) => write!(f, "{e}"),
```

Comment:
Nit: "io error: {e}"? Or else I guess this is equivalent to `e.fmt(f)`?

Reply:
Added a prefix.